src/pools/cluster/fixed.ts

```diff
@@ -36,16 +36,16 @@ export class FixedClusterPool<
   /**
    * Constructs a new poolifier fixed cluster pool.
    *
-   * @param numWorkers Number of workers for this pool.
+   * @param numberOfWorkers Number of workers for this pool.
    * @param filePath Path to an implementation of a `ClusterWorker` file, which can be relative or absolute.
    * @param opts Options for this fixed cluster pool. Default: `{ maxTasks: 1000 }`
    */
   public constructor (
-    numWorkers: number,
+    numberOfWorkers: number,
```

Comment:
same question as above

Reply:
Spell check is a good thing but we should keep backward compatibility as best as we can and I would prefer to not introducing anything that is not reaaaally beneficial if it breaks compatibility

Reply:
This is not a breaking change, because
1. its the v2 Cluster Pool
2. its a constructor parameter name